tests.gpr

```diff
@@ -0,0 +1,20 @@
+project Tests is
```

Comment:
Hope I'm not starting to sound like a broken record, but is it cool if these are using 3 spaces? I'll try and put something together for linting/lint-checking via ci soon.